python/astro_metadata_translator/translator.py

```diff
@@ -789,6 +789,24 @@ def to_exposure_group(self):
         else:
             return str(exposure_id)
 
+    @cache_translation
+    def to_observation_reason(self):
+        """Return the reason this observation was taken.
+
+        Base class implementation returns the ``science`` if the
+        ``observation_type`` is science, else ``calibration``.
+        A subclass may do something different.
+
+        Returns
+        -------
+        name : `str`
+            The reason for this observation.
+        """
+        obstype = self.to_observation_type()
+        if obstype == "science":
+            return "science"
+        return "calibration"
```

Comment:
I think I'd rather have "unknown" as the default non-science value.

Reply:
I'm fine with that. We do have the option of having explicit translations for HSC if you have some thoughts on what a reason should be there.